src/main/java/net/sf/jabref/gui/event/listener/TestEventListener.java

```diff
@@ -0,0 +1,19 @@
+package net.sf.jabref.gui.event.listener;
+
+
+import net.sf.jabref.gui.event.Event;
+
+public class TestEventListener implements EventListener {
```

Comment:
`TestEventListener` does not signal anything cause it implements the interface `EventListener`. `EventListener` signals with `@Subscribe` that it's accepting all events of type `Event` (subclasses included).